app/services/katello/pulp3/migration.rb

```diff
@@ -16,9 +16,21 @@ class Migration
         Katello::Erratum
       ].freeze
 
-      def initialize(smart_proxy, repository_types = REPOSITORY_TYPES)
+      def initialize(smart_proxy, repository_types = nil)
+        if repository_types.nil?
+          @repository_types = smart_proxy.supported_pulp_types[:pulp3][:overriden_to_pulp2]
```

Comment:
wouldn't this cause all types to be migrated?   Meaning if its set to migration only File and Docker (in the constant above), but yum is overriden to use pulp2, that'd get migrated too?

Reply:
Yeah, I'm missing a check for that.

Reply:
I pushed the new check.  The code should now error out if someone tries to create a migration service with un-migratable types in `:use_pulp_2_for_content_type`.